napari/layers/image/tests/test_big_image_timing.py

```diff
@@ -0,0 +1,32 @@
+import dask.array as da
+from napari.layers import Image
+import pytest
+
+
+data = da.random.random(size=(100_000, 1000, 1000), chunks=(1, 1000, 1000))
```

Comment:
@sofroniewn the array here should actually be a zarr array and not dask: dask always worked fine with `is_pyramid()`. If you use `zarr.zeros` it should not take up any memory.

Reply:
hmm I do rememeber that, I had just checked that both those fail with a big dask array too. I'll add zarr.zeros too. Nice to check both